package.json

```diff
@@ -75,12 +75,12 @@
         "when": "editorTextFocus && editorLangId == dafny"
       },
       {
-        "command": "dafny.showCounterExample",
+        "command": "dafny.showCounterexample",
```

Comment:
I worry that changing command names could break things for clients.  Could it?

Reply:
I won't break anything for the extension itself, and the way to enter commands in VSCode via CTRL+SHIFT+P is case and space insensitive anyway, so I wouldn't worry about it.

Reply:
Wouldn't it break custom shortcut configurations, for example? (I could be wrong here, I'm not familiar with vscode)

Reply:
It might, I haven't tested that, good point. But I don't see why someone would assign more shortcuts than F7 and F8, so it probably concerns no one.